lib/rules/selector-type-case/index.mjs

```diff
@@ -48,8 +49,7 @@ const rule = (primary, secondaryOptions) => {
 		}
 
 		root.walkRules((ruleNode) => {
-			let hasComments = ruleNode.raws.selector && ruleNode.raws.selector.raw;
-			const selector = hasComments ? hasComments : ruleNode.selector;
+			const selector = ruleNode.selector;
```

Comment:
question

Is this on purpose?
i.e. didn't you want `const selector = getRuleSelector(ruleNode);` instead?

Reply:
No, this is on purpose.

Here `ruleNode.selector` is only used for fast aborts.